src/components/Card.jsx

```diff
@@ -1,19 +1,28 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import { ThemeContext } from '../contexts/theme-context';
+import Popup from './Popup/Popup';
 
-export function Card({ card, index, markCard, deleteCard }) {
+export function Card({ card, index, markCard, renameCard, deleteCard }) {
```

Comment:
I think 1 callback that updates whole card better than few callbacks for each field

Reply:
done